# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

FtHoldActionGoal action_goal
FtHoldActionResult action_result
FtHoldActionFeedback action_feedback
